samples/rest/bing_auto_suggest_v7.py

```diff
@@ -0,0 +1,71 @@
+"""Python example showcasing the usage of Bing Auto Suggest API using REST calls"""
+
+import json
+from pprint import pprint
+
+import dotenv
+import requests
+
+
+def auto_suggest_basic(
+    subscription_key,
+    auth_header_name="Ocp-Apim-Subscription-Key",
```

Comment:
Note: if a var is constant across multiple files, it can be a config variable. You can create a config.py file, define the var and then all modules import from.
> no need to do here, just a note as it's only one variable.

Reply:
Every file inside 'samples/rest' is self-contained, this makes sense because the main goal for these files is for clients to mimic, I think introducing complexity is not the way to go.

Thanks for the `config.py` convention, Will most certainly use in SDKs.